### ISSUE TITLE
[Feature]: Inform the user why some commands are disabled

### ISSUE THREAD

**Issue by AlirezaHaghshenas** [ecadlabs/taqueria#1021]
### Describe the feature you would like

Commands can be disabled because

- A plugin is not installed
- A sandbox is not running
- Other minor reasons (for instance when all plugins are installed, install command will be disabled)

It's not easy for the user to know why a certain command is disabled and how they can make it available.

Suggested by @alexzbusko 

### Is this feature solving a problem?

Details about the problem your feature solves

### Please share any additional details you have?

Details about the problem your feature solves

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by russellmorton** [comment]
@AlirezaHaghshenas, where do users currently see that a command is disabled?

**Comment by AlirezaHaghshenas** [comment]
In the sidebar, for instance, if they have a sandbox, but Flextesa is not isntalled, the commands on that sandbox are disabled.
Also compile commands for contracts.

All commands in the sidebar except install/uninstall depend on a plugin and if that plugin is not installed, the command will be disabled.

The problem is, I'm afraid, we cannot change command tooltips dynamically.

**Comment by russellmorton** [comment]
Is it possible to have the buttons accept the click but then throw an error saying the plugin is not installed etc?

**Comment by AlirezaHaghshenas** [comment]
@russellmorton Yes. That's possible, but arguably it's a worse UX. What do you think?

**Comment by russellmorton** [comment]
This is a tough one, can we get in a huddle and talk about the options?
